solutions/vs2017/NUnit3CoreTestDemo/NUnit3CoreTestDemo.csproj

```diff
@@ -27,8 +20,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
-    <PackageReference Include="NUnit" Version="3.7.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
+    <PackageReference Include="NUnit" Version="3.9.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.9.0" />
```

Comment:
If we add the adapter as a package, doesn't that make it a bit harder to test new versions of the adapter? My strategy was to leave the package out so I could just add it in when testing a new build manually. If we add it, does that mean I will have to edit the csproj file in order to test a new version?

Reply:
Earlier we tested using the vsix adapter, that doesnt work for .net core, and we're more and more moving towards a nuget adapter approach.  Yes, we need to edit it to test a new version, but it is just a  copy/replace over the csproj files.   And, I think it is essential to test this with Visual Studio itself, not just the command line runner.